src/events.js

```diff
@@ -1429,6 +1429,8 @@ const collectionSubmission = async (participantData, biospecimenData, continueTo
             focus = false;
         }
 
+        if(hasError) return; // Do not set scanned ID if the tube has an error (further hardening for issues 1316/1388)
```

Comment:
```suggestion
        if (hasError) return; // Do not set scanned ID if the tube has an error (further hardening for issues 1316/1388)
```

Add space after `if` keyword for code styling convention.